test/shopify_app/carrier_service_verification_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+require 'action_controller'
+require 'action_controller/base'
+
+class CarrierServiceVerificationController < ActionController::Base
+  include ShopifyApp::CarrierServiceVerification
+  def carts_update
```

Comment:
can you rename this fake endpoint to something more realistic for this controller